src/analysis/normed_space/linear_isometry.lean

```diff
@@ -91,9 +95,9 @@ def to_continuous_linear_map : E →L[R] F := ⟨f.to_linear_map, f.continuous
 
 @[simp] lemma coe_to_continuous_linear_map : ⇑f.to_continuous_linear_map = f := rfl
 
-@[simp] lemma comp_continuous_iff {α : Type*} [topological_space α] {g : α → E} :
-  continuous (f ∘ g) ↔ continuous g :=
-f.isometry.uniform_embedding.to_uniform_inducing.inducing.continuous_iff.symm
+@[simp] lemma comp_continuous_iff {α : Type*} [topological_space α] {g : α → E₁} :
+  continuous (f₁ ∘ g) ↔ continuous g :=
+f₁.isometry.uniform_embedding.to_uniform_inducing.inducing.continuous_iff.symm
```

Comment:
Am I right that you don't actually need `E₁` here? You only need `f` to be `uniform_inducing`, and it is true. To prove this you need to replace `antilipschitz_with.uniform_embedding_of_injective` with `antilipschitz_with.uniform_inducing` and provide `isometry.uniform_inducing`.

Reply:
Do you mean to prove `antilipschitz_with.uniform_inducing`? I don't even know if this is true in general (for `pseudo_metric_space`)...

Reply:
It is true. I'm going to push this and a few other generalizations in an hour or two.

Reply:
I generalized some lemmas from `uniform_embedding` to `uniform_inducing`. This may break the build; fixes should be trivial - add `to_uniform_inducing` here and there.

Reply:
Thanks! Let's see if it compiles now.